openapi/openapi.yaml

```
@@ -95,6 +95,11 @@ tags:
     description: |-
       Use these operations to manage Rebilly accounts.
       An account in which you are subscribed to in order to use the Rebilly product.
+  - name: Allowlists
```

suggestion: I think we should call this feature `Allow lists`, similar to `value lists` 
```suggestion
  - name: Allow lists
```

---

We called it allowlists similar to blocklists: https://github.com/Rebilly/api-definitions/pull/1556